core/src/commonMain/kotlin/com/ustadmobile/core/torrent/UstadTorrentManager.kt

```diff
@@ -0,0 +1,17 @@
+package com.ustadmobile.core.torrent
+
+import org.kodein.di.DIAware
+
+interface UstadTorrentManager : DIAware {
```

Comment:
This needs some documentation.

e.g. explanation of what the TorrentManager is and how it fits in, and then a brief bit of documentation on each function.

```
/*
 * UstadTorrentManager manage torrent seeding and downloading. It is endpoint (context) specific. It will communicate with and manage the underlying torrent servers on the platform.
 */ 
```
Please add brief (e.g. 1-2 sentence) comments for each function